components/basics/react-native.js

```diff
@@ -60,6 +80,18 @@ const ReactNative = () => (
       the <Code>keyframes</Code> and <Code>injectGlobal</Code> helpers since React Native doesn't
       support keyframes or global styles. We will also warn you if you use media queries or nest your CSS.
     </p>
+
+    <p>
+      Since v2, <Code>css-to-react-native</Code> supports percentages. To make this possible we had to enforce unit usage when the value is a length.
+      See <Code><a href="https://github.com/styled-components/css-to-react-native/issues/20">css-to-react-native#20</a></Code> for more informations.
```

Comment:
Don't need a link to the issue, as the explanation fits in there and can be linked to from the codemod